docs/staff/editor/universe.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 2
+---
+
+# Universe
+
+The universe is where modules are displayed in a way that students can follow a certain direction or track.
+The role of the editor is to create these universes.
+
+## Viewing the universe
+
+When viewing the universe there are a couple of shortcuts and inputs to remember.
+
+| Action         | Result              |
+|----------------|---------------------|
+| R              | Reset view          |
+| CTRL + F       | Search for exercise |
+| Scroll         | Zoom in and out     |
+| Click and drag | Look around         |
+
+### Module attributes
+
+In the universe a module can have certain attributes.
+
+#### Certificate
+
+When a certificate is available for the module, it will be represented by a small yellow circle, with a certificate icon, on the top left of a module.
+
+![Module certificate](/img/staff/editor/universe/module-certificate.png)
```

Comment:
Dit stuk neem erg veel verticale ruimte op atm

Ik heb ff snel geprobeerd het in een table te zetten.
![image](https://github.com/BitAcademyDev/documentation/assets/1123154/4cbc1432-b488-48e6-bedf-01cf72ff8c89)

Wat vind je daarvan?

Reply:
Is wel mooier